.jest-global-setup.js

```diff
@@ -0,0 +1,3 @@
+module.exports = async () => {
+  process.env.TZ = 'UTC';
```

Comment:
Just curious, was this not something that would work in the existing `.jest-setup` file?

Reply:
Yep, for some I reason this needs to be in `globalSetup` part of `jest` setup, otherwise `process.env.TZ` has no effect here.